doc/source/management/changed_page_tracking.rst

```diff
@@ -4,7 +4,13 @@
 XtraDB changed page tracking
 =============================
 
-|XtraDB| now tracks the pages that have changes written to them according to the redo log. This information is written out in special changed page bitmap files.  This information can be used to speed up incremental backups using `Percona XtraBackup <http://www.percona.com/doc/percona-xtrabackup/>`_ by removing the need to scan whole data files to find the changed pages. Changed page tracking is done by a new |XtraDB| worker thread that reads and parses log records between checkpoints. The tracking is controlled by a new read-only server variable :variable:`innodb_track_changed_pages`.
+.. Important:: 
+
+        Starting with Percona Server for MySQL 8.0.26-17, the changed page tracking functionality is deprecated and may be removed in future versions.
+
+        We recommend using `MySQL Incremental Backup Using Page Tracking <https://dev.mysql.com/doc/mysql-enterprise-backup/8.0/en/mysqlbackup.incremental.html#mysqlbackup.page-tracking.incremental >`_ .
```

Comment:
Use this link instead of Incremental. We agreed, but I feel that users may be confused by sending them there.
[Page Tracking](https://dev.mysql.com/blog-archive/innodb-clone-and-page-tracking/)

Reply:
Fixed